test/release/examples/primers/fileIO.suppressif

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+
+import os
+
+platform = os.getenv('CHPL_TARGET_PLATFORM')
```

Comment:
Should these be using the scripts in `util/chplenv` instead of environment variables? I don't see anything that is setting these environment variables up.

Reply:
skipifs, supressifs, and other scripts like them are run by `./util/test/testEnv`, which injects the chplenv into the environment.

Reply:
Ah, I see. Thanks.